src/HonzaBotner.Discord/Command/Permissions/PermissionResolver.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.Entities;
+using HonzaBotner.Discord.Command.Permissions.Data;
+
+namespace HonzaBotner.Discord.Command.Permissions
+{
+    internal class PermissionResolver : IPermissionResolver
```

Comment:
Maybe sealed?

Reply:
Definitely not planning to extend this class, so I guess